website/www/site/content/en/blog/beamquest.md

```diff
@@ -0,0 +1,37 @@
+---
+title:  "Getting started with Apache Beam: An open source quest sponsored by Google Cloud"
```

Comment:
let's use a better SEO word here, like "credential" or "badge". 

(I feel like, developers would not search for "beam quests")